config.mak.uname

```diff
@@ -670,6 +670,7 @@ ifeq ($(uname_S),MINGW)
 	DEFAULT_HELP_FORMAT = html
 	HAVE_PLATFORM_PROCINFO = YesPlease
 	CSPRNG_METHOD = rtlgenrandom
+	LAZY_LOAD_LIBCURL = YesDoThatPlease
```

Comment:
It might make sense to mention in the commit message how/why this will help with our mainzenance.

Reply:
Sure, I can see that 😁